python/hail/dataset.py

```diff
@@ -3243,6 +3293,10 @@ def variant_qc(self):
         +---------------------------+--------+----------------------------------------------------+
         | ``dpStDev``               | Double | Depth standard deviation across all samples        |
         +---------------------------+--------+----------------------------------------------------+
+        | ``gqMean``                | Double | The average genotype quality across all samples    |
+        +---------------------------+--------+----------------------------------------------------+
```

Comment:
"Genotype quality mean across all samples" would be more parallel to the standard deviation one